index.js

```diff
@@ -74,6 +53,16 @@ class BabelCompiler {
     this.options.filename = file.path;
     this.options.sourceFileName = file.path;
 
+    if (!this.options.presets) {
+      const babelConfig = new babel.OptionManager().init(this.options);
+      console.log(babelConfig);
+      const hasConfig = babelConfig.presets && babelConfig.presets.length > 0;
+
+      if (!hasConfig) {
+        this.options.presets = ['latest'];
```

Comment:
@denysdovhan yeah, it could be so. but on the other hand, only 4 presets might be applied to this combination - 2015,16,17 and latest. but yeah, better just force people to use env with specified targets.